packages/dropdowns/src/elements/Dropdown/Dropdown.tsx

```diff
@@ -211,6 +215,10 @@ const Dropdown: React.FunctionComponent<IDropdownProps & ThemeProps<DefaultTheme
   );
 };
 
+Dropdown.defaultProps = {
+  role: 'listbox'
```

Comment:
I think `role` should be `undefined` by default. Rules:
- if provided by the user, then apply the user's value
- if used in the context of a `Select`, `Multiselect`, or `Autocomplete`, then apply `listbox`
- otherwise, apply a role of `menu`

Reply:
> if used in the context of a Select, Multiselect, or Autocomplete, then apply listbox

With `Dropdown` being instantiated before `Select/Multiselect/Autocomplete/Trigger` there may need to be some additional context work needed to apply the role.

Reply:
Exactly ☝️ This is what I meant earlier: https://github.com/zendeskgarden/react-components/pull/734#pullrequestreview-406784839

Reply:
It appears I misunderstood these comments. My understanding is that `Trigger` is only used in menu dropdowns, and that can be the heuristic to determine the appropriate ARIA attributes.

In the latest commit, I have removed all component `role` props. Consumers no longer need to pass in a `role`. The proper ARIA attributes are now determined by rendering a `Trigger` element, which will set a ref passed down from the dropdown context.